### ISSUE TITLE
About the classification loss is nan in retinaNet

### ISSUE THREAD

**Issue by wxr5899** [WZMIAOMIAO/deep-learning-for-image-processing#484]
**System information**
* Have I written custom code:
* OS Platform: Linux Ubuntu 16.04
* Python version: 3.7.11
* Deep learning framework and version(e.g., Tensorflow2.1 or Pytorch1.3): PyTorch 1.7.1
* Use GPU or not:
* CUDA/cuDNN version(if you use GPU):CUDA 11.0

大佬，我使用你的retinaNet代码，在训练VOC数据集的时候，出现了classification loss is nan的情况，我在加载数据集的代码中设置了检查bounding box是否合格的代码，并没有出现不合格的bounding box，而且不合格的box也只能导致 regression loss 出现问题。分类损失为nan是什么原因呢。

**Comment by WZMIAOMIAO** [comment]
你先用CPU训练试下看会不会出现nan的情况

**Comment by wxr5899** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by wxr5899** [comment]
谢谢大佬回复，问题已经解决了，是学习率的问题，调整学习率后训练就没有崩溃了
